common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/logging/DockerToActivationLogStore.scala

```diff
@@ -70,19 +72,37 @@ class DockerToActivationLogStore(system: ActorSystem) extends LogStore {
   override def fetchLogs(activation: WhiskActivation, context: UserContext): Future[ActivationLogs] =
     Future.successful(activation.logs)
 
+  def getLogs(transid: TransactionId,
```

Comment:
The name of this is awfully close to `fetchLogs` and `collectLogs`. Can we name this something like `logStream`? Also please make it protected, it's not supposed to be called from the outside. A piece of ScalaDoc and defining the return value would be the icing on the cake 🙂.

Defining the return value is a good best-practice for non-trivial methods. They make reviewing so much easier since we don't have an IDE in the Github review window just yet.

Reply:
we applied your proposals